mailbox/jpa/src/main/java/org/apache/james/mailbox/jpa/mail/JPAMessageMapper.java

```diff
@@ -385,6 +408,12 @@ protected MessageMetaData save(Mailbox mailbox, MailboxMessage message) throws M
         return data;
     }
 
+    private List<MessageUid> getUidList(List<MailboxMessage> messages) {
```

Comment:
Is this method used?

Reply:
It is, in `retrieveMessagesMarkedForDeletion`, line 207